### PR TITLE
[RHCLOUD-31282] Fix extra spacing on sections title and anchors

### PR DESCRIPTION
**Pull request by g-duval**
* Fix daily digest anchors
* Fix extra spacing on titles, introduced by <p> tag

### CODE REVIEW

File: engine/src/main/resources/templates/Common/insightsEmailBody.html
[the remark this answers is not in the document]
Comment (g-duval) [reply engine/src/main/resources/templates/Common/insightsEmailBody.html:1]: It's the same issue for all templates, but I can split and submit 2 PRs if you prefer ?

[the remark this answers is not in the document]
Comment (gwenneg) [reply engine/src/main/resources/templates/Common/insightsEmailBody.html:1]: No, one PR is fine. I just wanted to make sure all changes were intended.

File: engine/src/main/resources/templates/Common/insightsEmailBodyLight.html [hunk 1]
```diff
@@ -63,11 +63,12 @@
                     {/if}
                 </table>
             </td>
-        </tr>
+        </a>
```
Comment (gwenneg) [inline engine/src/main/resources/templates/Common/insightsEmailBodyLight.html:66]: This change looks wrong.

Comment (g-duval) [reply engine/src/main/resources/templates/Common/insightsEmailBodyLight.html]: Sure, thank you

**Review by gwenneg — APPROVED** [review]
LGTM

**Pull request merged**